build.gradle

```diff
@@ -109,6 +111,7 @@ plugins {
     id 'se.thinkcode.cucumber-runner' version '0.0.11' apply false
     id "com.github.davidmc24.gradle.plugin.avro-base" version "1.9.1" apply false
     id 'org.openapi.generator' version '7.8.0' apply false
+    id 'com.gradleup.shadow' version '9.0.0-beta4' apply false
```

Comment:
Having beta dependency is not nice. Would you give a try with latest stable version? 8.3.5?

Reply:
Downgraded to 8.3.5. Let us see